monitor.go

```diff
@@ -0,0 +1,48 @@
+package runc
+
+import (
+	"os/exec"
+	"syscall"
+)
+
+var Monitor ProcessMonitor = &defaultMonitor{}
+
+// ProcessMonitor is an interface for process monitoring
+//
+// It allows daemons using go-runc to have a SIGCHLD handler
```

Comment:
Maybe make it clear that the behavior of these methods should match that of the `exec.Cmd` type.